.github/workflows/base.yml

```diff
@@ -77,8 +77,27 @@ jobs:
           name: reports_dir
           path: ./docs/reports
 
+  run_stubtests:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}  # so that conda works
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install conda v3.7
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: '3.10'
```

Comment:
For some reason, stubtest cannot handle try/except imports, so i'm using Python 3.10 here.